content/en/agent/basic_agent_usage/_index.md

```diff
@@ -54,15 +52,16 @@ By default the Agent binds 3 [ports][3] on Linux and 4 on Windows and OSX:
 | 8125 | Used for the DogStatsD server to receive external metrics.                                  |
 
 ### The Collector
+
 The collector gathers all standard metrics every 15 seconds. Agent v6 embed a Python2.7 interpreter to run integrations and [custom checks][4].
```

Comment:
There's a typo in the last sentence: "Agent v6 embed". Should be "embeds". Also the statement is not 100% accurate since 6.14+ also embeds a 3.0 interpreter.